tests/modeladmin/tests.py

```diff
@@ -595,7 +595,7 @@ class ConcertAdmin(ModelAdmin):
         self.assertEqual(cmafa.base_fields['transport'].widget.attrs, {'class': 'radiolist inline'})
         self.assertEqual(
             list(cmafa.base_fields['transport'].widget.choices),
-            [('', 'None'), (1, 'Plane'), (2, 'Train'), (3, 'Bus')]
+            [(1, 'Plane'), (2, 'Train'), (3, 'Bus')]
```

Comment:
This change should not have impact on admin tests because in admin it already works properly (see [options.py](https://github.com/django/django/blob/661e6cc2c97d9bcb45198be787409488e1825c90/django/contrib/admin/options.py#L231-L234)).